request-handler/src/main/java/au/org/aodn/aws/wps/RequestHandler.java

```diff
@@ -1,7 +1,5 @@
 package au.org.aodn.aws.wps;
 
-import java.util.Properties;
-
 public interface RequestHandler {
-    AwsApiResponse handleRequest(AwsApiRequest request, Properties config);
+    AwsApiResponse handleRequest(AwsApiRequest request);
 }
```

Comment:
I'd leave this interface - the idea was that the properties could be rad from any source + passed in using the Properties object.